### PR TITLE
Add blurb and update about

### PR DESCRIPTION
**Pull request by dsudia**
@kytrinyx @masters3d 

Updates about information for prototype. Would love to get feedback/edits on the description. Tried to match the Go/R tracks as best I could in terms of style.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,223 +1,271 @@
 {
   "language": "TypeScript",
   "active": true,
+  "blurb": "TypeScript is a language for application-scale JavaScript. TypeScript adds optional types, classes, and modules to JavaScript.",
```
Comment (kytrinyx) [inline config.json]: What does application-scale mean? What problem with JavaScript does it solve?

File: config.json [hunk 2]
```diff
@@ -1,223 +1,271 @@
...
-      "core": false,
-      "unlocked_by": null,
+      "core": true,
       "difficulty": 1,
       "topics": [
-        "strings"
+        "Control-flow (conditionals)",
+        "Optional values",
+        "Strings",
+        "Text formatting"
```
Comment (kytrinyx) [inline config.json]: This PR includes the changes from the other one. Would you mind tweaking this so it only contains the copy changes?

File: docs/ABOUT.md [hunk 3]
```diff
@@ -1 +1,12 @@
-[TypeScript](http://www.typescriptlang.org/) is a language for application-scale JavaScript. TypeScript adds optional types, classes, and modules to JavaScript. TypeScript supports tools for large-scale JavaScript applications for any browser, for any host, on any OS. TypeScript compiles to readable, standards-based JavaScript. Try it out at the [playground](http://www.typescriptlang.org/Playground), and stay up to date via [our blog](https://blogs.msdn.microsoft.com/typescript) and [Twitter account](https://twitter.com/typescriptlang).
\ No newline at end of file
+[TypeScript](http://www.typescriptlang.org/) (TS) is a superset of JavaScript (JS), created at Microsoft
+in response to frustration developing large-scale applictions in JS.
```
Comment (kytrinyx) [inline docs/ABOUT.md]: Do you know what they were frustrated about?

File: docs/ABOUT.md [hunk 4]
```diff
@@ -1 +1,12 @@
...
+It is currently an open-source project hosted on Github.
+It supports tools for any browser as well as Node, for any host, on any OS. TTS compiles to readable, standards-based JavaScript.
+
+TS adds a flexible type system to JS, in addition to interfaces (custom types) and modifying the syntax of some ECMAScript features such as classes. It supports many upcoming
+ECMASCript features (such as async/await). Types allow tooling available in most code editors
+that improve the development experience such as code completion and method detection, both in
+your own code and in packages you use. It is compatible with all existing JS packages.
+TS transpiles to clean, readable JS.
+
+Try it out at the [playground](http://www.typescriptlang.org/Playground), and stay up to date via [our blog](https://blogs.msdn.microsoft.com/typescript) and [Twitter account](https://twitter.com/typescriptlang).
```
Comment (kytrinyx) [inline docs/ABOUT.md]: This is not Exercism's blog and Twitter account. Maybe _the Typescript blog and Twitter account_?

**Comment by dsudia** [comment]
@kytrinyx addressed your comments, let me know what you think. Thanks!

**Comment by kytrinyx** [comment]
Thanks so much @dsudia 🌻

**Pull request merged**